### ISSUE TITLE
Deploy multiple functions in the same pipe

### ISSUE THREAD

**Issue by yonner** [artemrys/google-cloud-functions-deploy#27]
Hi @artemrys really think this docker deploy script is great but wondering if it would be possible to pass / deploy multiple cloud functions in the same "-pipe" command currently i am deploying several functions in their own -pipe command i am wondering if the deployment process would be a lot more efficient if I could deploy all functions in the same docker image?

Is there any way to do that or would you need to make a modification to the script?

**Comment by artemrys** [comment]
Hi @yonner
Thanks for the feedback :)
Let me take a look at it this week and I'll get back to you

**Comment by yonner** [comment]
Hi just wondering if you found time to take a look at this?

**Comment by artemrys** [comment]
This will require a code change and some testing as well. I am not using this project myself anymore,  those changes I made before were easy and straightforward, this one will require more work and I do not have much time now to work on that.

**Comment by yonner** [comment]
Does that mean you're not doing further work on this? If so would it be difficult to take a fork of this and update myself, if so could you give my any pointers, it would be greatly appreciated.

**Comment by artemrys** [comment]
Yes, unfortunately I do not have time to properly maintain it and add new features.

So the script uses `gcloud` to perform all the actions (in `pipe/pipe.sh` file), it uses the following commands:

* `gcloud auth activate-service-account`
* `gcloud config set project`
* `gcloud functions deploy`

The `pipe/common.sh` file I just copied from other pipes built by Atlassian itself.

I also have a small testing repository in the Bitbucket itself to test the new versions of the pipe, it is really straightforward and you should know how to test it better than me 😄

You can also reuse the GitHub Action to release it both in Dockerhub and GitHub Packages located [here](https://github.com/artemrys/google-cloud-functions-deploy/blob/main/.github/workflows/release.yml). You may need to change my user (artemrys) to your user and provide secrets to GitHub.

**Comment by artemrys** [comment]
I just removed unnecessary file that were used to deploy using Bitbucket, but now it is on GitHub - I don't need that.

**Comment by artemrys** [comment]
Feel free to ping me here if you have more specific questions.
